Gems/EditorPythonBindings/Code/Source/Platform/Mac/PythonSystemComponent_mac.cpp

```diff
@@ -35,24 +35,24 @@ namespace Platform
         // append lib path to Python paths
         bool succeeded = true;
         
-        succeeded = succeeded && InsertPythonLibraryPath(paths, pythonPackage, engineRoot, "python/runtime/%s/Python.framework/Versions/3.7/lib");
+        succeeded = succeeded && InsertPythonLibraryPath(paths, pythonPackage, engineRoot, "python/runtime/%s/Python.framework/Versions/3.10/lib");
```

Comment:
Can we add compile definitions to the EditorPythonBidnings CMakeLists.txt, that has these paths in it using a PAL include?

Reply:
Actually I don't know why the original was targeting the specific version in the framework folder, when there is already a 'Current' symlink that will point to the correct one